internal/riverinternaltest/riverinternaltest.go

```diff
@@ -164,7 +164,7 @@ func DrainContinuously[T any](drainChan <-chan T) func() []T {
 func TestDB(ctx context.Context, tb testing.TB) *pgxpool.Pool {
 	tb.Helper()
 
-	ctx, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
It's so weird that this is not long enough. Are we creating enough test DBs for the `GOMAXPROCS`? They really shouldn't be waiting at all if so, at least not longer than whatever the time is to close the pool and wait for `TruncateRiverTables` 🤔

Reply:
Yeah it's so weird. This definitely merits deeper investigation, but ugh, don't feel like going down this rabbit hole right now. I was thinking the same around number of test DBs — seems like it's got to be something of that nature.